akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -54,22 +54,23 @@
         <maven-failsafe-plugin.version>3.1.2</maven-failsafe-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <maven-deploy-plugin.version>3.1.1</maven-deploy-plugin.version>
-
+        <akka-token>invalid-token</akka-token>
```

Comment:
@sebastian-alfers / @JustinPihony, would it be possible to fail the download with a clear message in case the passed token is `invalid-token`?